platform_versioned_docs/version-24.1/cli/commands.mdx

```diff
@@ -339,13 +344,17 @@ Run `tw compute-envs -h` to view the list of supported compute environment opera
   :::
 
   ```console
-  tw launch my_rnaseq_nf_pipeline 
+  tw launch my_rnaseq_nf_pipeline \
+  --config=<path/to/nextflow/conf/file> \
+
 
     Workflow 1XCXxX0vCX8xhx submitted at user workspace.
 
-      https://tower.nf/user/user1/watch/1XCXxX0vCX8xhx
+      https://cloud.seqera.io/user/user1/watch/1XCXxX0vCX8xhx
   ```
 
+  The optional `--config` flag is used to pass a custom Nextflow configuration file — configuration values passed here override the same values in the default pipeline `nextflow.conf`.
```

Comment:
The description is correct if the pipeline does not have a "nextflow config" field already set.
Whatever we add here will throw away everything from the "nextflow config" field from the pipeline and then it will override the same values in the defaults from the pipeline repo configuration.